dolphinscheduler-extract/dolphinscheduler-extract-base/src/main/java/org/apache/dolphinscheduler/extract/base/config/NettyClientConfig.java

```diff
@@ -32,7 +32,7 @@ public class NettyClientConfig {
      * worker threads，default get machine cpus
      */
     @Builder.Default
-    private int workerThreads = Runtime.getRuntime().availableProcessors() * 2;
+    private int workerThreads = Runtime.getRuntime().availableProcessors() * 2 + 1;
```

Comment:
May I ask why we have to `+1` here?

Reply:
SInce the thread will do some io task, and this is just an experience number.
You can get some from https://github.com/brettwooldridge/HikariCP/wiki/About-Pool-Sizing#the-formula.

Reply:
Thanks for your reply, my confusion is whether add one, default work thread in `EventLoopGroup` is `Runtime.getRuntime().availableProcessors() * 2`, it just assumes ratio of io-time and cpu-time is 1:1, and work thread equals `availableProcessors * (1 + 1/1)`. 

And in HikariCP's wiki, `+1` is for `effective_spindle_count` which may consider sth about disk IO.

Anyway, one more thread is not a big deal, just some personal confusion, if you think this setting is a more experienced value, I'm totally OK with that :D

Reply:
Thanks for point out this, I revert the count to 2 * core, keep the framework default is better.